lib/rspec/rails/swagger/request_builder.rb

```diff
@@ -47,6 +47,8 @@ def parameters location = nil
           else
             params
           end
+
+          params
```

Comment:
Seems like this would bypass the `location.present?` check. How were you running the command?